airflow/providers/google/provider.yaml

```diff
@@ -130,7 +130,7 @@ dependencies:
   - google-cloud-dataflow-client>=0.8.6
   - google-cloud-dataform>=0.5.0
   - google-cloud-dataplex>=1.10.0
-  - google-cloud-dataproc>=5.8.0
+  - google-cloud-dataproc>=5.10.1
```

Comment:
Why ?
Users can use 5.10 if they wish.
Why should we block older versions? 5.8 is not that old

Reply:
Yeah. Quite agree with Elad here. It might cause some conflicts if we are updating the minimum requirements "too aggressively".

Current constraints already point out to 5.10.1 https://github.com/apache/airflow/blob/constraints-main/constraints-3.10.txt#L341  and when people will be installing provider "with constraints" they will get it (they will also get it when installing provider without constraints). We have the "lowest dependencies" check running that currently verifies if we are not using - in our providers, some features that are not available in the "lowest bound" dependency versions - and we will bump minimum version when it finds that it should be bumped. There is no particular reason I think we should bump it manually here.

Reply:
Google Cloud Dataproc has added [](https://cloud.google.com/dataproc-serverless/docs/concepts/autotuning) and the API support for it is in version 5.10.1. We want our users to be able to set the cohort and scenarios on an out of the box.

Reply:
I am not sure if you understand @davidrabinowitz . There is nothing preventing your users from installing 5.10.1 now., in fact this is a default. Why do you want to forbid others to install previous version? What problem are you trying to solve?